themes/bubbles.omp.json

```diff
@@ -176,16 +177,6 @@
               "text": "\u276F",
               "postfix": ""
           }
-        },
-        {
```

Comment:
This was a weird, a coincidentally supported transparent text segment. Not intended to work and should break.